packages/types/src/user/service.ts

```diff
@@ -0,0 +1,35 @@
+import { Context } from "../shared-context"
+import { FindConfig } from "../common"
+import { IModuleService } from "../modules-sdk"
+import { FilterableUserProps, UserDTO } from "./common"
+import { CreateUserDTO, UpdateUserDTO } from "./mutations"
+
+export interface IUserModuleService extends IModuleService {
+  retrieveUser(
+    id: string,
+    config?: FindConfig<UserDTO>,
+    sharedContext?: Context
+  ): Promise<UserDTO>
+
+  listUsers(
```

Comment:
These are methods to retrieve and list from the main entity, I believe it is fine to keep them without the suffix (User).
`retrieve, list, listAndCount`

Reply:
Yes indeed, like the auto generated methods will be. I suggest you to glance over my pr description for the abstraction, i added all those conventions as well 👍

Reply:
We should probably await your PR before merging this one?